index.html

```diff
@@ -2838,24 +2818,28 @@ <h3><dfn>Writing or pushing content</dfn></h3>
     </section>
 
     <section><h3>Validating external type</h3>
-      <p>
+      <div>
         The [[NFC-RTD]] specifies that external types MUST contain
-        the domain name of the issuing organization, a colon and a type
-        name that is at least one character long.
-      </p>
-      <p>
-        The [[NFC-RTD]] specifies to omit the URN prefix “`urn:nfc:ext:`”
-        in the binary encoding of external type records, therefore
-        applications SHOULD NOT specify that when creating
-        <a>external type records</a>.
-      </p>
+        the domain name of the issuing organization, a colon (`:`) and a type
+        name that is at least one character long and follows the
+        <dfn>external type name ABNF</dfn>:
+        <pre class="abnf">
+          ext-type             = reg-name ":" custom-type
+          custom-type          = 1*(ALPHA / DIGIT / other)
+
+          DIGIT                = %x30-39
```

Comment:
Seems like NFC supports ip numbers :/ so maybe we need to use more from url spec (also that probably deserves a note)

Reply:
This definition is used only in the custom-type, so not.

Reply:
`reg-name` is not ABNF'd.

Reply:
but we have algorithm for checking it, so no problem IMHO